app/models/concerns/visible.rb

```diff
@@ -0,0 +1,18 @@
+module Visible
+  extend ActiveSupport::Concern
```

Comment:
Why did you create this concern? What is the benefit here?

Reply:
The benefit is that with a concern we can drive and share actions on many classes and create mixins to apply the DRY principle. In this case _visible_ will be used by __Article__ and __Comment__